### PR TITLE
Add a getCapabilities method to ngeo.Print

### PR DESCRIPTION
**Pull request by Jenselme**

### CODE REVIEW

File: src/services/print.js [hunk 1]
```diff
@@ -550,6 +549,24 @@ ngeo.Print.prototype.getReportUrl = function(ref) {
 
 
 /**
+ * Get the capabilities of the report
+ * @param {function()} success Success callback
+ * @param {function()=} opt_error Error callback
```
Comment (elemoine) [inline src/services/print.js]: These callbacks are not needed. The `getCapabilities` returns the http promise so the user can just call `success`, `error `or `then` on the promise.

**Comment by Jenselme** [comment]
> These callbacks are not needed. The getCapabilities returns the http promise so the user can just call success, erroror then on the promise. 

Indeed. Fixed (commits squashed).

File: src/services/print.js [hunk 2]
```diff
@@ -550,6 +549,19 @@ ngeo.Print.prototype.getReportUrl = function(ref) {
 
 
 /**
+ * Get the capabilities of the report
```
Comment (elemoine) [inline src/services/print.js]: Can you please just rephrase this to "Get the print capabilities from MapFish Print."? Once done I'll merge your branch. Thanks a lot for your contribution.

**Comment by Jenselme** [comment]
Done

File: src/services/print.js [hunk 3]
```diff
@@ -550,6 +549,19 @@ ngeo.Print.prototype.getReportUrl = function(ref) {
 
 
 /**
+ * Get the print capabilities from MapFish Print.
+ * @param {angular.$http.Config=} opt_httpConfig $http config object
+ * @return {Object}
```
Comment (elemoine) [inline src/services/print.js]: I've just realized that the return type is not correct. It should be `angular.$http.HttpPromise`.

File: src/services/print.js [hunk 4]
```diff
@@ -550,6 +549,19 @@ ngeo.Print.prototype.getReportUrl = function(ref) {
 
 
 /**
+ * Get the print capabilities from MapFish Print.
+ * @param {angular.$http.Config=} opt_httpConfig $http config object
```
Comment (elemoine) [inline src/services/print.js]: Please add a `.` at end of line.

**Comment by Jenselme** [comment]
> Please add a . at end of line.

Done

> I've just realized that the return type is not correct. It should be angular.$http.HttpPromise.

Corrected

**Comment by elemoine** [comment]
Cool, will merge when the build passed on Travis.
